.travis.yml

```diff
@@ -1,37 +0,0 @@
-sudo: required
-dist: trusty
-
-# trusty is used to let us use docker. This isn't possible in the container
-# based infrastructure. Docker is used to speed up the dapple installation,
-# which can run to ~4m using npm install.
-#
-# The trusty infrastructure does not allow caching, unlike the container based
-# default. This means waiting for npm install on each run. This may change in
-# the near™ future https://github.com/travis-ci/travis-ci/issues/4997
-
-language: node_js
-node_js: "6.8"
-
-services:
-  docker
-
-git:
-  submodules: false
-
-env:
-  - TEST=dapple
-
-before_install:
-  - id
-  - if [ $TEST = "dapple" ]; then npm install --progress=false -g dapple; fi
-  - if [ $TEST = "dapple" ]; then wget https://github.com/rainbeam/solidity-static/releases/download/v0.4.2-rel/solc; fi
-  - if [ $TEST = "dapple" ]; then sudo install solc /usr/local/bin/; fi
-  - cp .travis.dapplerc $HOME/.dapplerc
-
-install:
-  - if [ $TEST = "dapple" ]; then git submodule update --init --recursive; fi
-  - if [ $TEST = "npm"  ]; then npm set progress=false && npm install; fi
-
-script:
-  - if [ $TEST = "dapple" ]; then dapple test; fi
-  - if [ $TEST = "npm" ]; then npm test; fi
```

Comment:
I cringe when people use "continuous integration" to refer to automated builds, but you have a point